src/localization/de/learn.json

```diff
@@ -0,0 +1,163 @@
+
+  "examples.questionMark.title": "Übung: Fragezeichen `?`",
+  "examples.questionMark.description": "Schreiben Sie den Ausdruck, der anzeigt, dass der Buchstabe `n` im Text optional ist, indem Sie das Fragezeichen `?` verwenden. Somit können sowohl die Wörter `a` als auch `an` ausgewählt werden." ,
+
+  "examples.quantifier.title": "Übung: Geschweifte Klammern - 1",
+  "examples.quantifier.description": "Schreiben Sie den Ausdruck mit geschweiften Klammern `{}`, der Texte findet, die `4` Zahlen nebeneinander enthalten.",
+
+  "examples.quantifierMin.title": "Übung: Geschweifte Klammern - 2",
+  "examples.quantifierMin.description": "Schreiben Sie den Ausdruck mit geschweiften Klammern `{}`, der Texte findet, die mindestens `2` Zahlen nebeneinander enthalten.",
+
+  "examples.quantifierRange.title": "Übung: Geschweifte Klammern - 3",
+  "examples.quantifierRange.description": "Schreiben Sie den Ausdruck mit geschweiften Klammern `{}`, der Texte findet, die mindestens `1` und höchstens `4` Zahlen nebeneinander enthalten.",
+
+  "steps.groupping.title": "Klammern `( )`: Gruppierung",
+  "steps.groupping.description": "Wir können einen Ausdruck gruppieren und diese Gruppen verwenden, um einige Regeln zu referenzieren oder durchzusetzen. Um einen Ausdruck zu gruppieren, schließen wir `()` in Klammern ein. Fürs Erste gruppieren Sie einfach `haa` unten.",
+
+  "steps.groupReference.title": "Auf eine Gruppe verweisen",
+  "steps.groupReference.description": "Die Wörter `ha` und `haa` sind unten gruppiert. Die erste Gruppe wird verwendet, indem `\\1` geschrieben wird, um ein erneutes Schreiben zu vermeiden. Hier gibt `1` die Reihenfolge der Gruppierung an. Geben Sie ` ein \\2` am Ende des Ausdrucks, um auf die zweite Gruppe zu verweisen.",
+
+  "steps.nonCapturingGroupping.title": "Klammern `(?: )`: Gruppierung ohne Erfassung",
+  "steps.nonCapturingGroupping.description": "Sie können einen Ausdruck gruppieren und sicherstellen, dass er nicht durch Referenzen erfasst wird. Unten sehen Sie beispielsweise zwei Gruppen. Die erste Gruppenreferenz, die wir mit `\\1` bezeichnen, zeigt jedoch tatsächlich die zweite an Gruppe, da die erste eine nicht einfangende Gruppe ist.",
+
+  "steps.pipeCharacter.title": "Pipe-Zeichen `|`",
+  "steps.pipeCharacter.description": "Damit kann festgelegt werden, dass ein Ausdruck in verschiedenen Ausdrücken enthalten sein kann. Daher werden alle möglichen Anweisungen durch das Pipe-Zeichen `|` getrennt geschrieben. Dies unterscheidet sich vom Zeichensatz `[abc]`, Zeichensätze arbeiten auf Zeichenebene. Alternativen befinden sich auf Ausdrucksebene. Beispielsweise würde der folgende Ausdruck sowohl `Katze` als auch `Katze` auswählen. Fügen Sie am Ende des Ausdrucks ein weiteres Pipe-Zeichen `|` hinzu und geben Sie `Ratte` ein, damit alle Wörter werden ausgewählt.",
```

Comment:
This is the words intended to be in the text, so it should not be translated.
`katze`, `Katze`, `Ratte`

**Expected:** `cat`, `Cat`, `Rat`